diagnostics-app/src/app/diagnostics/diagnostics-dashboard/diagnostics-dashboard.component.html

```diff
@@ -1,7 +1 @@
-<!--
```

Comment:
Lets move this change to earlier, when we are creating the diagnostics-dashboard component.

Reply:
Done.